tests/_support/utils/router.js

```diff
@@ -0,0 +1,56 @@
+const _routes = new Map();
```

Comment:
Isn't this code in another PR somewhere? If not, maybe we should split it out into a separate PR